src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -555,26 +585,38 @@ internal bool IsInitialized(Guid guid)
         /// <returns>The custom node info object - null if we failed</returns>
         internal bool TryGetInfoFromPath(string path, bool isTestMode, out CustomNodeInfo info)
```

Comment:
I know you did not add this function, but this description seems wrong...

Reply:
Will Fix that